### PR TITLE
APP-6785: Remove local control page - remove web workflows

### PR DESCRIPTION
**Pull request by ethanlookpotts**
We are removing RC from the RDK - users are now expected to use app.viam.com to control their machine.

This PR just removes the workflows and anything web related from `.github`, which makes it easier to remove the code in the next step. (There is risk that there will be drift & issues in `web/frontend` until the latter is submitted, but we don't expect any changes there because we're removing it.)

1. 👉🏼 #4523
2. #4520 

## Review requests

- Check [`.github`](https://github.com/viamrobotics/rdk/tree/main/.github) - did I miss anything related to the web app?
- Are you the right person to review this PR? Is there someone else that you think should be on the PR?
- Does this plan work for you?

### CODE REVIEW

File: .github/workflows/full-static.yml [hunk 1]
```diff
@@ -45,8 +45,6 @@ jobs:
       with:
         go-version-file: go.mod
 
-    - name: build js
-      run: make build-web
```
Comment (ethanlookpotts) [inline .github/workflows/full-static.yml:49]: Is this safe for the time being? Or are the subsequent steps dependent on it?

Comment (abe-winter) [reply .github/workflows/full-static.yml:49]: guessing yes, safe

the makefile will automatically run `build-web` as a prereq of `make full-static` below

the reason I split it was for clarity around timing

Comment (ethanlookpotts) [reply .github/workflows/full-static.yml:49]: Awesome, thanks for clarifying!

**Review by abe-winter — APPROVED** [review]
lgtm

error cases here are:
- a workflow fails when you merge this
- a workflow fails after you merge the next PR

neither one drastic imo, and what you have looks right

**Review by zaporter-work — COMMENTED** [review]
Seems like there are a few other npm-publish.yml references:

I think this will cause the workflows on main to fail.
```
➜  workflows git:(APP-6785/ethanlookpotts/remove-web-workflows) ✗ rg "npm"
main.yml
63:  npm_publish:
64:    uses: viamrobotics/rdk/.github/workflows/npm-publish.yml@main

stable.yml
63:  npm_publish:
64:    uses: viamrobotics/rdk/.github/workflows/npm-publish.yml@main

releasecandidate.yml
46:  npm_publish:
47:    uses: viamrobotics/rdk/.github/workflows/npm-publish.yml@main
➜  workflows git:(APP-6785/ethanlookpotts/remove-web-workflows) ✗
```

**Comment by ethanlookpotts** [comment]
@zaporter-work great catch

[9984241](https://github.com/viamrobotics/rdk/pull/4523/commits/9984241074dc01cbe9fc75cbc63c0383a2249c22)

**Review by zaporter-work — APPROVED** [review]
Nothing else stands out -- LGTM

**Comment by ethanlookpotts** [comment]
Ah - didn't realize that James was OOO today - will wait to merge until tomorrow so he has a chance to review.

**Review by Otterverse — APPROVED** [review]
The workflow removals look fine to me. Bigger question about how this may break the agent's healtchecks (it queries localhost:8080 every minute) but will follow up about that in Slack/elsewhere.

**Comment by ethanlookpotts** [comment]
Holding on this PR until we resolve the discussion about agent healthchecks.

**Comment by ethanlookpotts** [comment]
I was able to verify that #4520 works as expected, so I'm going to go ahead and merge this and put the next PR up for review. Please holler if you notice anything with workflows!

**Pull request merged**